src/myrtlespeech/data/spec_augment.py

```diff
@@ -0,0 +1,447 @@
+# Copyright 2019 RnD at Spoon Radio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# Modified by Myrtle Software Ltd to merge files and remove all but required functionality
+# ==============================================================================
+import random
+
+import numpy as np
+import torch
+
+
+def spec_augment(
```

Comment:
Suggestion: put time warping in the backlog. The implementation looks slow and complex and it's not clear there's much to gain in the near-term. i.e. the risk of slowing down training time/adding bugs imposed by including this is greater than the gain it brings at this time. 

Just implementing time and frequency masking is super simple and can be extremely quick.